src/NuGet.Indexing/Sql2Lucene.cs

```diff
@@ -280,5 +284,30 @@ public static void Export(string sourceConnectionString, string destinationPath,
 
             stopwatch.Reset();
         }
+
+        private static DateTime GetCommitTimestampFromCatalog(Uri indexUrl, ILogger<Sql2Lucene> logger)
+        {
+            DateTime commitTime = DateTime.UtcNow;
+            try
+            {
+                using (var client = new System.Net.Http.HttpClient())
+                {
+                    var response = client.GetAsync(indexUrl).Result;
```

Comment:
nit: if this must be sync, make the method async and make the caller call `.Result` on the returned task. In other words, make as much of the stack async as possible.